app/pubspec.yaml

```diff
@@ -148,6 +148,7 @@ dependencies:
   open_filex: ^4.5.0
   phosphor_flutter: ^2.1.0
   device_calendar: ^4.3.2
+  quickalert: ^1.1.0
```

Comment:
Aren't we using ```EasyLoading``` package to show different data states? Why this one is needed on top??

Reply:
easyloading is transative, dialog-oriented and non-disruptive. quickalert provides further widget for more prominent, larger and more intrusive widgets. They are complementary.

Reply:
If that is so, do we want to consider unify all alerts be approached according to this package? If it also supports loading/error states?